.github/workflows/reusable_build.yml

```diff
@@ -0,0 +1,50 @@
+name: Functional tests using Ragger
+
+on:
+  workflow_call:
+    inputs:
+      uploaded_artifact_name:
+        required: true
+        type: string
+
+jobs:
+  build:
+    name: Build application for NanoS, X and S+
+    strategy:
+      matrix:
+        include:
+          - SDK: "$NANOS_SDK"
+            model: nanos
+          - SDK: "$NANOX_SDK"
+            model: nanox
+          - SDK: "$NANOSP_SDK"
+            model: nanosp
+          - SDK: "$STAX_SDK"
+            model: stax
+
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-lite:latest
+
+    steps:
+      - name: Clone
+        uses: actions/checkout@v3
+
+      - name: Build
+        run: |
+          git config --global --add safe.directory "$GITHUB_WORKSPACE"
+          make BOLOS_SDK=${{ matrix.SDK }}
+
+      - name: Get APPNAME value from Makefile
```

Comment:
it's not only getting APPNAME

Reply:
This step has been removed following the separate build mechanism